drivers/SRCommand.py

```diff
@@ -390,6 +392,12 @@ def run(driver, driver_info):
         else:
             print(ret)
 
+    except ProtocolError:
+        # xs_errors.__new__ returns a different class
+        # pylint: disable-msg=E1101
+        print(xs_errors.XenError(
```

Comment:
You may not have anyone to report to, e.g. if someone did a toolstack restart and get EPIPE on this probably.
Can you also log it to a file like with other exceptions?

Reply:
Done